dawn/setup.py

```diff
@@ -0,0 +1,169 @@
+##===-----------------------------------------------------------------------------*- Python -*-===##
+##                          _
+##                         | |
+##                       __| | __ ___      ___ ___
+##                      / _` |/ _` \ \ /\ / / '_  |
+##                     | (_| | (_| |\ V  V /| | | |
+##                      \__,_|\__,_| \_/\_/ |_| |_| - Compiler Toolchain
+##
+##
+##  This file is distributed under the MIT License (MIT).
+##  See LICENSE.txt for details.
+##
+##===------------------------------------------------------------------------------------------===##
+
+import os
```

Comment:
There are some more subtle differences, but to sum up: a bare `import name` only makes the `name` symbol available in the current module, so if you want to use an object of function of the module, you need to prefix it as `name.useful_function`. If you use `from root_name.sub_name import more_useful_function` then `more_useful_function` is directly available in the module without prefixing. In general this is useful when the imported useful is used quite a lot in the module or if the prefix is very long. In this case, only the symbols coming from the `setuptools` and `distutils` modules are imported directly, because the prefixes are kind of long and the overall installation script is slightly easier to read, but it is mostly a matter of style.